spec/spec_helper.rb

```diff
@@ -0,0 +1,46 @@
+# encoding: utf-8
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/inputs/zeromq"
+require "support/client"
```

Comment:
We should use `require_relative` here, the require will fail when running the tests on jenkins (the all plugins test).
This is because the support folder isn't added to the path.